src/SourceBuild/tarball/content/Directory.Build.props

```diff
@@ -247,6 +247,9 @@
     <!-- if MicrosoftSourceLinkCommonPackageVersion is non-empty, then we've already built SourceLink, regardless of whether
          this is the production or offline build, so we should use that version.  -->
     <ExtraPackageVersionPropsPackageInfo Include="MicrosoftSourceLinkVersion" Version="%24(MicrosoftSourceLinkCommonPackageVersion)" />
+
+    <!-- Used by runtime to determine current Roslyn version -->
+    <ExtraPackageVersionPropsPackageInfo Include="MicrosoftCodeAnalysisVersion_4_X" Version="%24(MicrosoftCodeAnalysisVersion)" />
```

Comment:
Why doesn't this belong in `repos/runtime.common.props`? are there other repos that also need this prop?

Reply:
This was an oversight on my part.  I thought the ExtraPackageVersionPropsPackageInfo infra was scoped to this props file.  I see other repos are utilizing it.  I will move this as suggested.